### ISSUE TITLE
Additional information for Docker Desktop / WSL2 users on windows for mounting the data files

### ISSUE THREAD

**Issue by designzwang** [Overv/openstreetmap-tile-server#220]
Hi,

I was not able to run the container in import mode, and it took me quite some time to find out what was going wrong.
In my opinion, the README should contain hints how to mount the data files when using Docker Desktop / WSL2.

I wasn't shure whether to create a pull request for the README for this or not - if want me to do that, I will. Just let me know if you see fit.

Looking at the container logs, I could see that osm2pgsql could not read the mounted data file /data.osm.pbf, because no matter what I tried it was always mounted as an (empty) directory.

In Power Shell / Windows Terminal, I followed the instructions in the README and tried 

`docker run  -e UPDATES=enabled     -v ${PWD}/datafile.osm.pbf:/data.osm.pbf     -v ${PWD}/polyfile.poly:/data.poly   -v openstreetmap-data:/var/lib/postgresql/12/main    overv/openstreetmap-tile-server import  `

Even though in this shell ${PWD}/datafile.osm.pbf was accessible ( `dir ${PWD}/datafile.osm.pbf` works and shows the file ) , the file was not accessible for docker and so it was not possible to mount it in the container.

I tried all variations of path syntax that come to mind :
 - /mnt/c/path/to/pbf-file
 - /c/path/to/pbf-file
 - /path/to/pbf-file
 - c:\path\to\pbf-file
 - ${pwd}/pbf-file

None of these worked, the result was , in all cases,  two empty directories /data.poly and /data.osm.pbf. 

Eventually, I ran the docker command from within the WSL shell, i.e. in the default WSL distro , and there it worked with both
 ${PWD} and  /mnt/c/... 

`user@wslshell$  docker run  -e UPDATES=enabled     -v /mnt/c/path/to/datafile.osm.pbf:/data.osm.pbf     -v /mnt/c/path/to/polyfile.poly:/data.poly   -v openstreetmap-data:/var/lib/postgresql/12/main    overv/openstreetmap-tile-server import  `

For WSL2, Docker and MS recommend to store data that to be shared inside the linux file system, see https://docs.docker.com/desktop/windows/wsl/#best-practices 

However, since the PBF files could be quite big and should probably not be copied/moved into the WSL VHD, it does also work  when they are in the windows file system and mounted with their /mnt/c/.... path  . This might be slower, but since the import is done only once, i don't see a problem. 

So, to sum it up, it might be sufficient to simply add the recommendation to the README that WSL users should run the docker command from the WSL shell.

Regards,
Ulrich
